src/main/java/athleticli/storage/Config.java

```diff
@@ -0,0 +1,8 @@
+package athleticli.storage;
+
+/**
+ * Defines string literals or configurations used for file storage.
+ */
+public class Config {
+    public static final String PATH_SAVE = "./data/athleticli.txt";
```

Comment:
This is not a plain text file, so the `.txt` extension should be avoided.
```suggestion
    public static final String PATH_SAVE = "./data/athleticli.bin";
```